bb.edn

```diff
@@ -0,0 +1,62 @@
+{:min-bb-version "0.8.0"
+ :pods {;since babashka 0.8.0, can only be declared in bb.edn
+        org.babashka/tools-deps-native {:version "0.0.5"}   ;; for babashka build
+        clj-kondo/clj-kondo            {:version "2022.10.05"}}
+ :deps  {datahike/bb {:local/root "bb"}}
+ :tasks {; tools
+
+         fmt {:doc "Format source files"
+              :task tasks.format/fix}
+
+         docs {:doc "Get a preview on the clojure docs"
+               :task tasks.cljdoc/preview}
+
+         ; checks
+
+         fmt? {:doc "Test formatting"
+               :task tasks.format/check}
+
+         lint {:doc "Run linter"
+               :task tasks.lint/-main}
+
+         test {:doc "Run all tests or restrict to 'unit', 'integration' or 'back-compat'"
+               :task tasks.test/-main}
+
+         kaocha {:doc "Run kaocha with arbitrary arguments"
+                 :task tasks.test/kaocha}
+
+         deps {:doc "Check consistency of dependencies"
+               :task tasks.deps/check}
+
+         check {:doc "Run all checks"
+                :depends [test fmt? lint deps]}
+
+         ; build and release
+
+         settings {:doc "Show content of settings file"
+                   :task tasks.settings/show}
+
+         version {:doc "Set or increment the project version [major|minor] <version>"
+                  :task tasks.version/-main}
+
+         tag {:doc "Return current version as a tag"
+              :task tasks.version/as-tag}
+
+         clean {:doc "Remove build files" :task tasks.build/clean}
+
+         compile {:doc "Compile classes" :task tasks.build/compile}
+
+         pom {:doc "Create pom file" :task tasks.build/pom}
+
+         jar {:doc "Build jar" :task tasks.build/jar}
+
+         build {:doc "Process files and build jar" :task tasks.build/package}
+
+         install {:doc "Install jar locally"
```

Comment:
This call does not find the `pom.xml`, though the pom command created one in a `/target` subfolder.

Reply:
there was some issue with reading the edn, so the path was a bit weird. Should be fixed now.